cvmfs/cloud_testing/run.sh

```diff
@@ -17,35 +18,12 @@ script_location=$(cd "$(dirname "$0")"; pwd)
 . ${script_location}/common.sh
 
 config_package_base_url="https://ecsft.cern.ch/dist/cvmfs/cvmfs-config"
-
-# static information (check also remote_setup.sh and remote_run.sh)
-cvmfs_workspace="/tmp/cvmfs-test-workspace"
-cvmfs_source_directory="${cvmfs_workspace}/cvmfs-source"
-cvmfs_log_directory="${cvmfs_workspace}/logs"
-
-# global variables for external script parameters
-testee_url=""
-client_testee_url=""
-platform=""
-platform_run_script=""
-platform_setup_script=""
-ec2_config=""
-ami_name=""
-log_destination="."
-username="root"
-userdata=""
-
-# package download locations
-server_package=""
-client_package=""
-devel_package=""
-config_packages=""  # might be more than one... BEWARE OF SPACES!!
-unittest_package=""
-source_tarball="source.tar.gz" # will be prepended by ${testee_url} later
```

Comment:
What I mean is: Those variables are not explicitly set inside this script but they "somehow" come from the outside. In this case they are `export`-ed by the calling script (`run_cloudtests.sh`). Don't they?

Passing variables around like that is fine in general, but at the moment it's unclear that those variables are coming from the outside. By convention global variables that come from "somewhere" (aka. environment variables) should be named in capitals and with a namespace prefix. I'd just reuse `CT_` here as well, since it's part of the `CloudTesting` infrastructure.

Readability is key here (because we have a ton of interoperating shell scripts that make it easy to loose track), and variables that are "somehow" coming from the caller is a lot of fairy dust...